lib/cli_helper.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'open3'
+
+module CliHelper
+  class << self
+    # Execute a Terraform command with open3.
+    # Supported option is `:dir` which is optionally the path to the Terraform project dir
+    def execute(command, opts = {})
+      if opts[:dir]
+        begin
+          Open3.capture3(command, chdir: opts[:dir])
+        rescue Errno::ENOENT
+          msg = if File.directory?(opts[:dir])
+                  "Could not find executable 'terraform'"
```

Comment:
I'm vaguely concerned that there's some other way this error will be raised, and this message won't be correct. Can we just pass through the `e.message` from the error - for this case at least it's pretty sane: `No such file or directory - terraform`